resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -186,15 +186,15 @@
       {% if useERCC %}
         {% for exp in cuffquant %}
           S_NAME={{ exp|sample_name }}
-          parse_cuffnorm.py cuffnorm_all_fpkm_exprs_norm.txt {{ loop.index0 }} -n
+          parse_cuffnorm.py cuffnorm_all_fpkm_exprs_norm.txt ${S_NAME} -n
```

Comment:
I think this should be in quotes: `"${S_NAME}"` to cover the case where sample name contains whitespace